src/kirby/components/chart/options/donut.ts

```diff
@@ -10,6 +10,8 @@ export const DonutOptions: Options = {
     },
     backgroundColor: 'transparent',
     type: '',
+  },
+  accessibility: {
```

Comment:
This looks bad (for accessibility)... won't people with sight disabilities have a problem figuring out what the graph is (based on this option)?